helper_test.go

```diff
@@ -1189,11 +1191,14 @@ func createWorkspaceWithOptions(t *testing.T, client *Client, org *Organization,
 
 	return w, func() {
 		if err := client.Workspaces.Delete(ctx, org.Name, w.Name); err != nil {
-			t.Errorf("Error destroying workspace! WARNING: Dangling resources\n"+
-				"may exist! The full error is shown below.\n\n"+
-				"Workspace: %s\nError: %s", w.Name, err)
+			if errors.Is(err, ErrResourceNotFound) {
```

Comment:
Could you explain this change?

Reply:
So one of the integration tests was failing (checking for removed workspace). It was due to the test cleanup failing; errored trying to remove a workspace that was already removed.

So it begged the question; If you try to cleanup a workspace that's already been cleaned up, should it throw an err?

---

However this all moot because I've just noticed the test could be setup in a way I didn't have to do this change. But the question is still valid, and perhaps it should be in a different PR.

Reply:
Fixed up

Reply:
@glennsarti That's a good question re what to do with a workspace if it's already been cleaned up. I've opened an internal issue for the team to discuss this and update our contributing docs with some guidance.